be-java/src/commonMain/kotlin/lang/temper/be/java/RunJava.kt

```diff
@@ -151,8 +153,22 @@ internal fun runJavaBestEffort(
             false -> when (runJava) {
                 is RunByExec -> {
                     // Currently always for a specific library.
-                    val libDir = relativeOutputDirectoryForLibrary(specifics.backendId, runLibrary!!)
-                    listOf(ToolchainResult(libraryName = runLibrary, doRunJava(libDir.resolve(runDir))))
```

Comment:
Note this old code did backend id then library *then* runDir, which is the opposite order from above. So presumably this could only work for empty paths, which I'm pretty sure we always got.